Streetcode/Streetcode.XIntegrationTest/ControllerTests/Utils/BeforeAndAfterTestAtribute/Streetcode/ExtractTestUpdateStreetcode.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Streetcode.DAL.Entities.Streetcode;
+using System.Reflection;
+using Xunit.Sdk;
+
+namespace Streetcode.XIntegrationTest.ControllerTests.Utils.BeforeAndAfterTestAtribute.Streetcode
+{
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false, Inherited = false)]
+    class ExtractTestUpdateStreetcode : BeforeAfterTestAttribute
+    {
+        public static StreetcodeContent StreetcodeForTest;
+
+        public override void Before(MethodInfo methodUnderTest)
+        {
+            var sqlDbHelper = BaseControllerTests.GetSqlDbHelper();
+            StreetcodeForTest = sqlDbHelper.GetExistItem<StreetcodeContent>();
+            if (StreetcodeForTest == null)
+            {
+                StreetcodeForTest = sqlDbHelper.AddNewItem(new StreetcodeContent()
+                {
+                    Index = new Random().Next(0, 1000000),
+                    UpdatedAt = DateTime.Now,
+                    CreatedAt = DateTime.Now,
+                    EventStartOrPersonBirthDate = DateTime.Now,
+                    EventEndOrPersonDeathDate = DateTime.Now,
+                    ViewCount = 1,
+                    DateString = "20 травня 2023",
+                    Alias = "dsf",
+                    Title = "Title",
+                    TransliterationUrl = Guid.NewGuid().ToString(),
+                    Teaser = "Test Teaser",
+                });
+                sqlDbHelper.SaveChanges();
+            }
```

Comment:
I suggest populating variable `StreetcodeForTest` of type **StreetcodeUpdateDTO** with data in `Before` method
#### Example:
``` csharp
StreetcodeForTest = CreateMoqStreetCodeDTO(
                _streetcodeContent.Id,
                _streetcodeContent.Index,
                _streetcodeContent.TransliterationUrl);
```
P.S. Here I assume that we have private function `CreateMoqStreetCodeDTO` in `ExtractTestUpdateStreetcode`;